src/database.js

```diff
@@ -1,14 +1,22 @@
 import { Sequelize } from 'sequelize'
 
+const setDatabase = () => {
+  let database
+  if (process.env.DATABASE_URL) {
+    database = process.env.DATABASE_URL
+  } else {
+    database = 'postgres://postgres:postgres@localhost/postgres'
+  }
+  return database
+}
+
 const Connection = new Sequelize(
-  'postgres',
-  'postgres',
-  'postgres',
+  setDatabase(),
```

Comment:
this can be condensed to: `process.env.DATABASE_URL || 'postgres://postgres:postgres@localhost/postgres'` like we do for the port here https://github.com/esarafianou/harmonise.it/blob/a85bd7/src/server.js#L27